backend/src/main/kotlin/rocks/didit/sefilm/web/controllers/ShowingController.kt

```diff
@@ -125,9 +125,35 @@ class ShowingController(private val repo: ShowingRepository,
     }
 
     val participantsInfo = participantRepo.findByShowingId(showing.id)
+    updateFöretagsbiljettAsUsed(showing.participants)
     return BuyDTO(shuffledBioklubbnummer(showing), sfLink, participantsInfo, showing.participants)
   }
 
+  private fun updateFöretagsbiljettAsUsed(participants: Set<Participant>) {
+    val relevantUsers = participants.filter { p ->
+      when(p) {
```

Comment:
hela lambdan borde du kunna byta ut mot `it.payment.type == PaymentType.Företagsbiljett`